README.md

```diff
@@ -881,6 +881,29 @@ interface FCPAttribution {
 #### `INPAttribution`
 
 ```ts
+/**
+ * Summary information about the slowest script intersecting the INP duration
+ * as provided by the Long Animation Frame API.
+ *
+ * NOTE: Only scripts above 5 milliseconds are included in long animation
+ * frames.
+ */
+export interface longestScriptSummary {
```

Comment:
```suggestion
interface LongestScriptSummary {
```

Interface names should be capitalized. Also, for the README I've been omitting the `export` keywords, to keep things as simple as possible in the docs.